git_big_picture/_main.py

```diff
@@ -498,13 +498,12 @@ def write_to_file(output_file, dot_output):
 
     """
     try:
-        f = open(output_file, 'wb+')
+        with open(output_file, 'wb+') as f:
+            f.write(dot_output)
+            f.flush()
```

Comment:
Quick question, are the flush and fsync needed, even if we are doing this inside a context manager?

Reply:
It *may* not be needed, but:
- I can only "guarantee" a call to `open` as a context manager calls `.close`, nothing more
- We may have had some good reason to add it back then (needs a closer look), and before being sure, we may produce a regression.

@esc so unless you can prove these calls as no longer needed, my vote is for not dropping them due to risk and low current pain.

PS: From https://docs.python.org/3/library/io.html#io.TextIOWrapper:
> [IOBase](https://docs.python.org/3/library/io.html#io.IOBase) is also a context manager and therefore supports the [with](https://docs.python.org/3/reference/compound_stmts.html#with) statement. In this example, file is closed after the with statement’s suite is finished—even if an exception occurs:

Reply:
OK, it's not worth the effort, let's just keep them and be safe.